lib/trento/application/integration/checks/checks.ex

```diff
@@ -17,16 +17,15 @@ defmodule Trento.Integration.Checks do
   @spec request_execution(String.t(), String.t(), ClusterExecutionEnv.t(), [map], [String.t()]) ::
           :ok | {:error, :any}
   def request_execution(execution_id, cluster_id, env, hosts, selected_checks) do
-    execution_requested =
-      ExecutionRequested.new!(
-        execution_id: execution_id,
-        group_id: cluster_id,
-        targets:
-          Enum.map(hosts, fn %{host_id: host_id} ->
-            Target.new!(agent_id: host_id, checks: selected_checks)
-          end),
-        env: build_env(env)
-      )
+    execution_requested = %ExecutionRequested{
```

Comment:
The same as the other contract PR. Shouldn't we go with `struct` function to have a 1:1 replacement, and most probably apply validation.